src/react/deckgl.js

```diff
@@ -119,7 +119,7 @@ export default class DeckGL extends React.Component {
 
   // Route events to layers
   _onClick(event) {
-    const {x, y} = event;
+    const {event: {offsetX: x = 0, offsetY: y = 0} = {}} = event;
```

Comment:
Just a sanity check. 

I agree it makes more sense to drop the event if either `event.event` is falsy (luma.gl addEvent fails, shouldn't be) or `offsetX` `offsetY` are falsy (browser fails, again shouldn't be according to the [spec](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/offsetX)); it's also heavy and requires changes on existing apps if we surface the error message via the picked object.

will just drop the event if sanity check fails.